workspace/test_injection.sh

```
@@ -0,0 +1,148 @@
+        # Add headers required for SharedArrayBuffer (Thread support)
+        self.send_header("Cross-Origin-Opener-Policy", "same-origin")
+        self.send_header("Cross-Origin-Embedder-Policy", "require-corp")
+        super().end_headers()
+
+with socketserver.TCPServer(("", PORT), Handler) as httpd:
+    print(f"🚀 Game server running! Open http://localhost:{PORT} in your browser.")
+    print("Press Ctrl+C to stop the server.")
+    httpd.serve_forever()
+EOF
+
+cd export/web
```

Shellcheck (0.11.0)

[warning] Use 'cd ... || exit' or 'cd ... || return' in case cd fails.

(SC2164)